gradle_test.go

```diff
@@ -80,7 +80,7 @@ func TestGradleBuildWithServerIDWithUsesPlugin(t *testing.T) {
 }
 
 func TestGradleBuildWithCredentials(t *testing.T) {
-	if *tests.RtUser == "" || *tests.RtPassword == "" {
+	if *tests.RtAccessToken != "" {
```

Comment:
All build tools, including gradle should support access tokens. Wht are we then skipping this test here?

The same question goes for the TestMavenBuildWithCredentials function.